### PR TITLE
Add script to capture Webpack bundle analysis in CI

### PR DESCRIPTION
**Pull request by spadgett**
We might want to block merges if the main vendors chunk gets too large, but just capture the report for now.

/cc @alecmerdler

### CODE REVIEW

**Comment by spadgett** [comment]
https://github.com/openshift/release/pull/4555 enables this in CI

**Comment by spadgett** [comment]
/hold

Let me check there's no issue with this. The report is ~1.7MB.

**Comment by alecmerdler** [comment]
Should we get rid of the old bundle analysis? Looks like the [artifact is still there](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_console/2185/pull-ci-openshift-console-master-e2e-aws-console/5756/artifacts/e2e-aws-console/gui_test_screenshots/bundle-analysis.txt) but has the wrong data.

**Comment by spadgett** [comment]
/hold cancel

Removed the previous bundle analysis

**Comment by spadgett** [comment]
/assign @jcaianirh 

@jcaianirh ptal

File: frontend/integration-tests/tests/performance.scenario.ts [hunk 1]
```diff
@@ -33,18 +31,6 @@ const chunkedRoutes = OrderedMap<string, {section: string, name: string}>()
 
 describe('Performance test', () => {
 
-  it('checks bundle size using ResourceTiming API', async() => {
```
Comment (alecmerdler) [inline frontend/integration-tests/tests/performance.scenario.ts]: Don't we still want this? It captures the actual chunks received by the browser.

Comment (spadgett) [reply frontend/integration-tests/tests/performance.scenario.ts]: Is it working? I thought the bundle-analysis file always 0 and you asked me to remove :)

File: analyze.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+ARTIFACT_DIR=${ARTIFACT_DIR:=/tmp/artifacts}
```
Comment (jcaianirh) [inline analyze.sh]: @spadgett do you need to set this again? Isn't this already specified when ci calls the script?

`  commands: export ARTIFACT_DIR=/tmp/artifacts; ./analyze.sh`

Comment (spadgett) [reply analyze.sh]: It will use the value from the environment. This defaults it if not set.

**Comment by spadgett** [comment]
Rebased

File: frontend/integration-tests/tests/performance.scenario.ts (hunk 1 above)
Comment (spadgett) [reply frontend/integration-tests/tests/performance.scenario.ts]: Circling back on this... This is the test that wrote `bundle-analysis.txt` to the filesystem and always returned 0. I think having the full report is much more valuable regardless.

I haven't removed the tests for the various routes below.

**Comment by spadgett** [comment]
@jcaianirh can you lgtm if this looks ok? :)

**Comment by jcaianirh** [comment]
/lgtm
sorry missed this request

**Pull request merged**